views/partials/foot.jade

```diff
@@ -0,0 +1,4 @@
+if scripts && scripts.length
+	each script in scripts
+		if !script.inHead
+			script(async=script.async crossorigin=script.crossorigin defer=script.defer integrity=script.integrity src=script.src text=script.text type=script.type)
```

Comment:
It shouldn't, jade should strip out all undefined properties
`<script crossorigin="anonymous" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" src="//code.jquery.com/jquery-3.2.1.slim.min.js"></script>`